flight/Modules/Airspeed/airspeed.c

```diff
@@ -359,22 +361,25 @@ void baro_airspeedGet(BaroAirspeedData *baroAirspeedData, uint32_t *lastSysTime,
 }
 #endif
 
-static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev)
+static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev, void *ctx, void *obj, int len)
 {
-	AirspeedSettingsData airspeedSettingsData;
-	AirspeedSettingsGet(&airspeedSettingsData);
-	
-	airspeedSensorType=airspeedSettingsData.AirspeedSensorType;
-	gpsSamplePeriod_ms=airspeedSettingsData.GPSSamplePeriod_ms;
+	(void) ev; (void) ctx; (void) obj; (void) len;
```

Comment:
This is to mark the variables as "used"-- doing nothing with them-- so you don't get a warning about them being unused -- an alternative to __attribute__((unused)) in the declaration.  Our build flags don't currently complain about unused parameters but I wanted to avoid making them worse.